src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/landing_page_template.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: 'Landing page template'
+tags:
+translate:
+redirects:
+---
+
+Landing pages are a specialized type of page that serve as the starting pages for various New Relic products. For example, you'll see landing pages for **Application monitoring (APM)** and **Browser monitoring**. 
+
+<Callout variant="tip">
+  
+  This landing page information does not apply to the docs home page.
+  
+</Callout>
+
+
+If you need to create a new landing page, you can either copy an existing landing page, or you can modify the sample landing page shown at the bottom. Let's look at the sections you need for each landing page.
+
+## Front matter
+
+When you insert the front matter, be sure to designate the `type` as `landingPage`. Here's an example:
+
+```
+---
+title: New Relic APM
+type: landingPage
+tags:
+translate:
+redirects:
```

Comment:
Not sure how you want to communicate this, but these fields are optional and can be omitted if they are blank. It does make the frontmatter a bit cleaner.

That being said, these frontmatter fields are common amongst all page types. I'm not sure if you have another section talking about these, but at the very least, it might be nice to mention these are optional and can be omitted.

Reply:
@jerelmiller, I'll find where we document front matter and make sure we mention that "tags," "translate," and "redirects" are optional. Is "title" the only required element? What about type (for example "landingPage") and when is it required?

Reply:
Unfortunately we don't have great documentation on that (yet) so I can't give you a good reference to look.

To quickly answer your last question:

> Is "title" the only required element? What about "type" (for example "landingPage") and when is it required?

`title` is the only required field. If `type` is omitted, then the page is treated as a generic "doc page" (basic page in drupal), which means it gets the right rail, default "For more help" section, etc. You only need `type` when you're authoring a page that isn't a generic doc page (i.e. `landingPage` as you mentioned, `releaseNote`, etc)

We'll make a note that we need to get you the list of supported frontmatter fields. Some frontmatter items are supported by all docs (tags, redirects), where some are specific to other types of content `releaseDate` on `releaseNote` for example).

Reply:
@jerelmiller, what about "template" in the front matter? This appears to be optional, but we should also list the available options.

Reply:
`template` is optional and will default to `docsPage` if not provided (the basic template type). Since these are landing pages, the `template` **MUST** be `landingPage` in order to get this page to look right. I just read the doc and looks like you have this correct!